<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transWindow</class>
 <widget class="QMainWindow" name="transWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="transfer">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Meltix Regular Demo</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Transfer</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFrame" name="frame1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="Bverify">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
  border-radius:20px;
  margin: 0;

</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBgetCard">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Meltix Regular Demo</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <item>
               <property name="text">
                <string>Choose Card</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Gown">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>45</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Meltix Regular Demo</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:transparent;
     border-radius: 20px;
     border: 1px solid ;
     width: 20px;
     height: 1px;
     padding-left: 10px;
border-top-color: transparent;
border-right-color: transparent;
border-left-color: transparent;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="arrow">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Gother">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>45</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Meltix Regular Demo</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:transparent;
     border-radius: 20px;
     border: 1px solid ;
     width: 20px;
     height: 1px;
     padding-left: 10px;
border-top-color: transparent;
border-right-color: transparent;
border-left-color: transparent;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLineEdit" name="Gamount">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Meltix Regular Demo</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:transparent;
     border-radius: 20px;
     border: 1px solid ;
     width: 20px;
     height: 1px;
     padding-left: 10px;
border-top-color: transparent;
border-right-color: transparent;
border-left-color: transparent;</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="placeholderText">
            <string>Amount(Rials)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>60</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="Gstatic">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>45</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Meltix Regular Demo</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:transparent;
     border-radius: 20px;
     border: 1px solid ;
     width: 20px;
     height: 1px;
     padding-left: 10px;
border-top-color: transparent;
border-right-color: transparent;
border-left-color: transparent;</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BvisiableStat">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">
background-color: transparent;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Bmessage">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/new/prefix1/email.jpg);
background-color: transparent;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Gdynamic">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>45</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Meltix Regular Demo</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>IBeamCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:transparent;
     border-radius: 20px;
     border: 1px solid ;
     width: 20px;
     height: 1px;
     padding-left: 10px;
border-top-color: transparent;
border-right-color: transparent;
border-left-color: transparent;</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BvisiableDy">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">
background-color: transparent;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="Bback">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>60</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Meltix Regular Demo</family>
               <pointsize>12</pointsize>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">margin-top: 50px;
color: rgb(85, 255, 255);
background-color: rgb(0, 0, 0);
  text-align: center;
  border-radius:30px;
box-sizing: border-box;
  margin: 0;
border-color: rgb(0, 255, 255);
  padding: 0;</string>
             </property>
             <property name="text">
              <string> Back</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Btransfer">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>60</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Meltix Regular Demo</family>
               <pointsize>12</pointsize>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">margin-top: 50px;
color: rgb(85, 255, 255);
background-color: rgb(0, 0, 0);
  text-align: center;
  border-radius:30px;
box-sizing: border-box;
  margin: 0;
border-color: rgb(0, 255, 255);
  padding: 0;</string>
             </property>
             <property name="text">
              <string> Transfer</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="otherName">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Meltix Regular Demo</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 170, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Error">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Meltix Regular Demo</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
